src/components/ui/invoice/GenerateInvoice.tsx

```diff
@@ -60,10 +61,13 @@ const GenerateInvoice = () => {
       (sum, entry) => sum + entry.amount * entry.quantity,
       0,
     );
-    setTotalAmount(subtotal);
+    setTotalAmount(FormatToUSD(subtotal));
 
     const totalTax = taxDetails.reduce((sum, tax) => sum + tax.percentage, 0);
-    setTotalWithTax(subtotal + (subtotal * totalTax) / 100);
+    // Calculation of Total amount with taxes
+    const TotalWithTaxAmount = subtotal + (subtotal * totalTax) / 100
```

Comment:
`T` should be in small case

Reply:
Hey, changes made again. please take a look and let me know your ideas.